podfile

```diff
@@ -1,4 +1,4 @@
-platform :ios, '13.0'
+platform :ios, '14.0'
 target 'ArcGIS Runtime SDK Samples' do
   use_frameworks!
   pod 'ArcGIS-Runtime-Toolkit-iOS', '~> 100.14'
```

Comment:
Should we bump this up to `100.14.1`? https://cocoapods.org/pods/ArcGIS-Runtime-SDK-iOS

Reply:
Ahh yes, forgot about that

Reply:
I tried but it said "could not apply suggestion" 😢